lmms.1

```diff
@@ -54,15 +54,21 @@ Specify output bitrate in kHz (for OGG encoding only), default is 160
 Specify interpolation method - possible values are \fIlinear\fP, \fIsincfastest\fP (default), \fIsincmedium\fP, \fIsincbest\fP
 .IP "\fB\-x, --oversampling\fP \fIvalue\fP
 Specify oversampling, possible values: 1, 2 (default), 4, 8
+.IP "\fB\-l, --loop
+Render the given file as a loop, i.e. stop rendering at exactly the end of the song. Additional silence or reverb tails at the end of the song are not rendered.
 .IP "\fB\-u, --upgrade\fP \fIin\fP \fIout\fP
 Upgrade file \fIin\fP and save as \fIout\fP
 .IP "\fB\-d, --dump\fP \fIin\fP
 Dump XML of compressed file \fIin\fP (i.e. MMPZ-file)
 .IP "\fB\-v, --version
 Show version information and exit.
+.IP "\fB\    --allowroot
+Bypass root user startup check (use with caution).
 .IP "\fB\-h, --help
 Show usage information and exit.
 .SH SEE ALSO
+.BR http://lmms.io/
+.BR https://github.com/LMMS/lmms
 .BR http://lmms.sf.net/
 .BR http://lmms.sf.net/wiki/
```

Comment:
Any reason to keep `http://lmms.sf.net` around even though it redirects to `https://lmms.io`, which is also listed?